lib/index.js

```diff
@@ -10,10 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* globals process, console, Q, JSON, escape, verifyToken, crypto */
+/* globals process, console, SDK_VERSION */
 /*jshint maxcomplexity:15 */
 
-var SDK_VERSION = '<%= sdkversion %>'; // Will be replaced by grunt
+require('./vendor/polyfills');
```

Comment:
This is the only sane way I could find to inject the polyfills. The general consensus online is to define what your environment expects and then force your consumers to polyfill the environment themselves. We don't want this for our customers. I'll add a comment to that affect after you guys review.